web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -204,11 +203,87 @@ class ComboMarkdownEditor {
     });
   }
 
-  prepareEasyMDEToolbarActions() {
+  prepareEasyMDEToolbarActions(EasyMDE) {
     this.easyMDEToolbarDefault = [
-      'bold', 'italic', 'strikethrough', '|', 'heading-1', 'heading-2', 'heading-3', 'heading-bigger', 'heading-smaller', '|',
-      'code', 'quote', '|', 'gitea-checkbox-empty', 'gitea-checkbox-checked', '|',
-      'unordered-list', 'ordered-list', '|', 'link', 'image', 'table', 'horizontal-rule', '|', 'clean-block', '|',
+      {
+        name: 'heading',
+        action: EasyMDE.toggleHeading3,
+        icon: svg('octicon-heading'),
+        title: 'Heading',
```

Comment:
Could we find some h1-h3 icons for the editor? I guess it might be useful.

Reply:
Sorry, but I'm not motivated right now for that 😆

Reply:
Leave the wiki editor toolbar broken?

Reply:
I guess it could keep the string name, and replace them with Gitea's definition.

Reply:
Good news is that we do not need any new icon.

```
.editor-toolbar button.heading-1::after {
    content: "1";
}
.editor-toolbar button::after {
    font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
    font-size: 65%;
    vertical-align: text-bottom;
    position: relative;
    top: 2px;
}
```

Reply:
Fixed:

<img width="273" alt="image" src="https://user-images.githubusercontent.com/115237/235750316-4c4ed7d7-af18-4ea9-a991-85760a14511f.png">